### PR TITLE
Edit destination functionality

### PR DESCRIPTION
**Pull request by vTrip**
Implemented the popover onClick of edit button

### CODE REVIEW

File: www/css/ionic.app.css [hunk 1]
```diff
@@ -9798,7 +9798,8 @@ a.button {
       font-size: 12px;
       text-align: left; }
       .trip-detail .destination-item .row .departure-date-label label .depart-text {
-        font-size: 10px; }
+        font-size: 10px;
```
Comment (jsuresh) [inline www/css/ionic.app.css]: THis file shoouldn't be in git, please remove

File: www/css/ionic.app.css [hunk 2]
```diff
@@ -9798,7 +9798,8 @@ a.button {
...
+        line-height: none; }
```
Comment (jsuresh) [inline www/css/ionic.app.css]: ditto

File: www/js/controllers.js [hunk 3]
```diff
@@ -238,6 +238,22 @@ angular.module('awayapp')
     });
   }
 
+  var changeDatePopover = $ionicPopover.fromTemplateUrl('templates/destination-date-popover.html', {
+    scope: $scope,
+    backdropClickToClose: true
+  }).catch(function(err) {
+    //TODO(vince// any error handling here...?)
```
Comment (jsuresh) [inline www/js/controllers.js]: This is popping up a lot, can you add a ticket to trello saying we need to have a (default) unified method of handling errors.

**Comment by jsuresh** [comment]
LGTM, please address comments and merge in :)

File: www/css/ionic.app.css (hunk 1 above)
Comment (vTrip) [inline www/css/ionic.app.css]: when i delete from git - is it 'npm install ionic' to get the ionic.app*.css files back?

**Comment by vTrip** [comment]
Would prefer if I didn't merge until with you seeing as git states there are conflicts and I want to do the correct process to resolving them and then merging

**Pull request merged**